CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 5.5.1 (2021-01-21)
+
+* Alter ANR SIGQUIT handler to stop interfering with Google's ANR handler
+  [#1078](https://github.com/bugsnag/bugsnag-android/pull/1078)
```

Comment:
This also fixes the async issue raised in issue 1026 - would be good to reference this in the changelog entry somewhere.

Reply:
Reworded to add this to the description